src/Apps/Artist/Routes/Consign/Components/__tests__/ArtistConsignMeta.test.tsx

```diff
@@ -0,0 +1,99 @@
+import { MockBoot } from "DevTools"
+import { mount } from "enzyme"
+import React from "react"
+import { ArtistConsignMeta } from "../ArtistConsignMeta"
+
+jest.mock("Utils/getENV", () => ({
+  getENV: () => "https://artsy.net",
+}))
+
+describe("ArtistConsignMeta", () => {
+  const props = {
+    artistName: "Alex Katz",
+    artistHref: "/artist/alex-katz",
+    imageURL: "path/to/image.jpg",
+  }
+  const getWrapper = (passedProps = {}) => {
+    return mount(
+      <MockBoot>
+        <ArtistConsignMeta {...props} {...passedProps} />
+      </MockBoot>
+    )
+  }
+
+  it("outputs correct title tags", () => {
+    const wrapper = getWrapper()
+    expect(wrapper.find("Title").debug()).toContain(props.artistName) // use `debug` to assert on component tree, vs html tree
```

Comment:
I don't think this is super important, but since it's a text element, you could do `wrapper.find("Title").text()` to assert against a more direct/less noisy value.

Reply:
So with the title tag, it actually does some magic in the background to inject a `<title>` meta tag into the header. Since we're not rendering the full page, and thus don't have access to stuff in the `<header>`, we can use `debug` to assert on the react tree itself. This provides sufficient assurances.